tv_apps/smart-system/style/zindex.css

```diff
@@ -239,11 +239,11 @@
   z-index: 20;
 }
 
-#screen > [data-z-index-level="app"] > .appWindow.transition-opening:not(.homescreen):not(.attentionWindow) {
+#screen > [data-z-index-level="app"] > .appWindow.transition-closing:not(.attentionWindow) {
   z-index: 18;
 }
 
-#screen > [data-z-index-level="app"] > .appWindow.transition-closing:not(.homescreen):not(.attentionWindow) {
+#screen > [data-z-index-level="app"] > .appWindow.transition-opening:not(.attentionWindow) {
   z-index: 16;
 }
 
```

Comment:
This is for showing the home fade-out animation when we launch an app that is already in background